docs/expr-lang.ebnf

```diff
@@ -140,3 +141,12 @@ numeric_literal = ? clang::tok::numeric_constant ? ;
 boolean_literal = "true" | "false" ;
 
 pointer_literal = "nullptr" ;
+
+builtin_func = builtin_func_name "(" {builtin_func_argument_list} ")" ;
+
+builtin_func_name = "__log2" ;
+
+builtin_func_argument_list = builtin_func_argument
+                           | builtin_func_argument_list "," builtin_func_argument_list
```

Comment:
One of those builtin_func_argument_lists could be builtin_func_argument, no?

Also, is there a reason to forbid empty argument list?

Reply:
(Looking at the parser, it looks like non-empty arguments was an intentional decision.)